runtime-display/x11-lib/autobuild/build-order

```diff
@@ -1,35 +1,35 @@
-xtrans-1.4.0.tar.gz
```

Comment:
Consider rewriting this as you would for x11-app/font?

Reply:
Done.